pkg/ipamd/ipamd.go

```diff
@@ -380,7 +380,8 @@ func (c *IPAMContext) nodeInit() error {
 		for {
 			retry++
 
-			if err = c.setupENI(eni.ENIID, eni, eni.ENIID == metadataResult.TrunkENI, metadataResult.EFAENIs[eni.ENIID]); err == nil {
+			if c.awsClient.TagENI(eni.ENIID, metadataResult.TagMap[eni.ENIID]) == nil &&
```

Comment:
Since VPC CNI doesn't manage Trunk ENIs, maybe we can skip default tagging for these ENIs.

Reply:
make sense. Our IPAMD currently do "manage" Trunk ENIs somehow, e.g. reconcile the SGs to be same as primary ENI's SGs.
However, it do feels odd to tag the ENIs created by vpc-resource-controller from IPAMD. 
I'll skipped the tagging for trunk ENIs. And customers using trunk ENI can to add additional iam permission to allow CNI manage these trunk ENIs. (trunk ENIs are tagged with `kubernetes.io/cluster/<clusterName>: owned` on creation.

In the future, we can change vpc-resource-controller to tag trunk ENIs with `cluster.k8s.amazonaws.com/name: <cluster-name> as well.